initfiles/componentfiles/config2xml/esp.xsd

```diff
@@ -20,10 +20,10 @@
                                             <xs:attribute name="description" type="xs:string" use="optional" hpcc:displayName="Description"/>
                                             <xs:attribute name="path" type="xs:string" use="required" default="/" hpcc:displayName="Path" hpcc:tooltip="The logical path of a resource used for authentication"/>
                                             <xs:attribute name="resource" type="xs:string" use="required" hpcc:displayName="Resource" hpcc:tooltip="The physical resource for which access is checked"/>
-                                            <xs:attribute name="access" default="Read">
+                                            <xs:attribute name="access" hpcc:codeDefault="Read">
```

Comment:
I thought this attribute name was going to change to defaultValueInCode or something to that affect

Reply:
I was going to do that as the subject of a separate PR. If you want it done on this PR, it's not really related to the subject of the PR.

Reply:
That's fine, if there's no Jira for it yet, let's make sure we open one.

Reply:
HPCC-19978 opened for this puprpose